source/guide_lychee.rst

```diff
@@ -0,0 +1,88 @@
+.. highlight:: console
+
+.. sidebar:: Logo
+  
+  .. image:: _static/images/lychee.png 
+      :align: center
+
+#########
+Lychee
+#########
+
+Lychee_ is a open source photo-management software written in PHP and distributed under the MIT license. It allows you to easily upload, sort and manage your photos, all while presenting those with a beautiful web interface.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of 
+
+  * PHP_
+  * MySQL_ 
+  * domains_
+
+Prerequisites
+=============
+
+We're using PHP_ in the stable version 7.1:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version show php
+ Using 'PHP' version: '7.1'
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Also, the domain you want to use for Lychee must be set up as well:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+``cd`` to your `document root`_, then clone the latest release of the Lychee installer and move the files to the document root:
+
+::
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ git clone https://github.com/electerious/Lychee.git
```

Comment:
This way you pull the master branch. I think something like 
```
[isabell@stardust html]$ git clone --branch release/42.13.2 https://github.com/electerious/Lychee 
```
would be better. Then add a 

```
----
Tested with Lychee 3.1.6, Uberspace 7.1.1
```
to the [end of your guide](https://github.com/Uberspace/lab/blob/master/source/README.md)

Reply:
Thanks a lot for the feedback @nichtmax . I added the version specific git commands and improved the rest as suggested.